ios/Podfile.lock

```diff
@@ -586,7 +586,7 @@ SPEC CHECKSUMS:
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   FBLazyVector: 49cbe4b43e445b06bf29199b6ad2057649e4c8f5
-  FBReactNativeSpec: 091e3ebb4e41b04653769e622c0dc90f8670318e
+  FBReactNativeSpec: cfcd14f56ff0e9e07e965d363a0fb0384dc6a5ec
```

Comment:
Is this change necessary? Sorry, I really don't know

Reply:
not necessary, because every new build generates new podfile hashes